### PR TITLE
implement scheduled transfers/orders getters

### PR DESCRIPTION
**Pull request by YasseinBilal**

### CODE REVIEW

File: src/common/queries/scheduledOrders.ts [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    },
+    body: JSON.stringify({
+      query,
+      variables,
+    }),
+  })
+
+  const responseBody = await response.json()
+
+  console.log('responseBody', responseBody)
```
Comment (kopy-kat) [inline src/common/queries/scheduledOrders.ts:42]:
```suggestion
```

File: src/common/queries/scheduledOrders.ts [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+  if (response.ok) {
+    console.log('responseBody.data', responseBody.data)
```
Comment (kopy-kat) [inline src/common/queries/scheduledOrders.ts:45]:
```suggestion
```

File: src/common/queries/scheduledOrders.ts [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+  } else {
+    throw new Error(
+      `Error: ${responseBody.errors.map((error: any) => error.message).join(', ')}`,
+    )
+  }
+}
+
+const queryById = `
+    query ($jobId: String) {
+      scheduledOrdersExecutionAddedQueries (where:{ jobId: $jobId}) {
```
Comment (kopy-kat) [inline src/common/queries/scheduledOrders.ts:56]: this also requires the smartaccount right?

File: src/common/queries/scheduledTransfers.ts [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+  } else {
+    throw new Error(
+      `Error: ${responseBody.errors.map((error: any) => error.message).join(', ')}`,
+    )
+  }
+}
+
+const queryById = `
+    query ($jobId: String) {
+      scheduledTransfersExecutionAddedQueries (where:{ jobId: $jobId}) {
```
Comment (kopy-kat) [inline src/common/queries/scheduledTransfers.ts:53]: same as above

File: src/common/queries/scheduledOrders.ts (hunk 3 above)
Comment (YasseinBilal) [reply src/common/queries/scheduledOrders.ts]: Yep, Fixed

File: src/common/queries/scheduledTransfers.ts (hunk 4 above)
Comment (YasseinBilal) [reply src/common/queries/scheduledTransfers.ts]: Fixed

**Pull request merged**